execution.go

```diff
@@ -258,7 +269,8 @@ func (q *queryExecution) createGQLErrors(step *QueryPlanStep, err error) gqlerro
 
 	var gqlErr GraphqlErrors
 	var outputErrs gqlerror.List
-	if errors.As(err, &gqlErr) {
+	switch true {
```

Comment:
```suggestion
	switch {
```
You don't need the true if you're using the case statement as an if/else?